frontend/src/pages/blogs.js

```diff
@@ -0,0 +1,84 @@
+import { LayoutDefault } from "layouts";
+import { NextSeo } from "next-seo";
+import { PageHeader, Pager } from "shared-components";
+import ENV from "config/env";
+import { BlogList } from "components";
+import BlogsService from "./api/blogsService";
+import { useState } from "react";
+import { BlogFilters } from "../components/blog-filters/blog.filters";
+
+const { BASE_URL = "", BASE_SEO = "", STATIC_DIR = "", AUTHOR } = ENV;
+
+function Blogs(props) {
+	const {
+		pathname,
+		pageSize,
+		data: {
+			title,
+			items,
+			pagination,
+			metaDescription,
+		},
+	} = props;
+
+	const SEOS = {
+		title,
+		description: metaDescription,
+		canonical: `${BASE_URL}${pathname}`,
+		openGraph: [
+			{
+				url: BASE_URL,
+				images: { url: `${BASE_URL}${STATIC_DIR}logo.png` },
+				site_name: AUTHOR,
+			},
+		],
+		...BASE_SEO,
+	};
+	const [activePageNumber, setActivePageNumber] = useState(1);
+	const [blogs, setBlogs] = useState(items);
+	const [numberOfPages, setNumberOfPages] = useState(pagination.total_pages);
+
+	async function getBlogs(pageNumber, contains = null, category = null) {

```

Comment:
Je l postoji neki razlog sto ovde contains i category assignujes na null kao param?

Reply:
Iskreno, nisam probao bez. U Python-u je to nacin da se stavi default vrednost argumenta funkcije sto omogucava da se funkcija pozove bez tih argumenata koji imaju default vrednost. Sada vidim da u JS to jednostavno bude `undefined` (kada ne stavis default vrednost) sto je isto ok.